src/Api/ApiClient.php

```diff
@@ -192,6 +203,8 @@ public function request(
 			$data
 		);
 
+		$this->logRequest($url, $queries, $data, $response);
```

Comment:
`$url` je třeba předávat tu co přijde argumentem, před aplikací queries. A oříznout k prvnímu výskytu `{`. Jinak se u get requerů loguje url `payment/status/M1E3CB0860/f7af7de3fc0d1BF/20160628124720/mRHrFwN4FY8C0X37f%2BeNSGqfMDvBWVwMZNibMGBj%2Bfpo9Tx05BCFddtyGShQlfog91lRgMccSN6XJL8HyLi2PdPg2Gfq6mD8hBcYjXbHI0pRG9PB2R7v1hKtBUwOBMI0sEzdOjP%2BVRZHS%2FDBP5P0KhB3OgvWFTbJ5GsNGaBk8P4TYKL%2BqdhXZA8FKFy%2F1Fc%2Ff25EJF%2FaAdjEEiKWIjBfyTP7WH9FiIXW8tal0R2vLJuxcVMfx%2FiY2hFDnqax7aq0d2qG0SMcs4TUGm%2F77SxOUrHudfeuS0FsVqEdsVqbXclQ56FN4D967IajxJXrNrxlN3X2YgVATlk5fa8%2BPTHcVA%3D%3D` a to nechcem